src/index.js

```diff
@@ -16,13 +16,15 @@ import {
 } from 'react-router-dom';
 
 import { Security } from '@okta/okta-react';
```

Comment:
Can this line be removed since the line below replaces it?

Reply:
Absolutely! I will remove this line and the wrapper around the components as it has been replaced with Auth0 ProtectedRoute